From cb734964028230718ac1a9e305691c3a04a223ce Mon Sep 17 00:00:00 2001
From: Zhe Zhang <zhezhang@cloudera.com>
Date: Fri, 27 Feb 2015 11:42:58 -0800
Subject: [PATCH 411/596] CLOUDERA-BUILD. Add proto-web.xml which is part of
 HADOOP-10563. This is required in order to build
 the backport of HDFS-7424 (Add web UI for NFS
 gateway)

---
 .../hadoop-hdfs/src/main/webapps/proto-web.xml     |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/proto-web.xml

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/proto-web.xml b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/proto-web.xml
new file mode 100644
index 0000000..83b36b2
--- /dev/null
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/webapps/proto-web.xml
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+<web-app version="2.4" xmlns="http://java.sun.com/xml/ns/j2ee">
+</web-app>
-- 
1.7.9.5

